dev-docs/RFCs/v6.1/project32-with-64bit-data-rfc.md

```diff
@@ -0,0 +1,42 @@
+# RFC: Projection Mode Improvements
+
+* **Authors**: Ib Green and Georgios Karnas
+* **Date**: June 2019
+* **Status**: Draft
+
+Notes:
+
+## Summary
+
+This RFC proposes adding a new LNG/LAT projection mode that consumes 64bit input data but then uses 32 bit linearized calculations for 90% of 32 bit performance with 90% of 64 bit precision.
```

Comment:
Don't understand the 90%

Reply:
Just trying to convey that this projection mode should be "almost as fast as" the standard 32 bit projection, while offering "almost the same precision as" full 64 bit projection.